### ISSUE TITLE
Add E-mail generator to contrib script

### ISSUE THREAD

**Issue by phrocker** [apiri/utils#1]
Would be neat if it generated the E-mail for you. Perhaps as a gmail plugin?

**Comment by apiri** [comment]
Yeah, haven't gotten to the level of plugin, but did the semi-automatic generation as part of the release helper script that is now woefully outdated: https://gist.github.com/apiri/857bd858498ec345a664be740e7ca40b

**Comment by phrocker** [comment]
That's some impressive bash
